projects/map-ol/src/lib/map-ol.component.ts

```diff
@@ -234,11 +234,27 @@ export class MapOlComponent implements OnInit, AfterViewInit, AfterViewChecked,
       case GeojsonLayertype:
         this.updateGeojsonLayerParamsWith(oldLayer as any, newLayer as VectorLayer);
         break;
+      case WfsLayertype:
+        this.updateWfsLayerParamsWith(oldLayer as any, newLayer as VectorLayer);
+        break;
       default:
         break;
     }
   }
 
+  updateWfsLayerParamsWith(oldLayer: olVectorLayer, newLayer: VectorLayer) {
```

Comment:
If we want to refactor `@dlr-eoc/map-ol` later https://github.com/dlr-eoc/ukis-frontend-libraries/issues/72 then maybe we should mark this function as private for now.

Reply:
done!